queue_services/entity-filer/tests/unit/__init__.py

```diff
@@ -360,66 +360,66 @@
 }
 
 INCORP_FILING = {
-        'filing': {
-            'header': {
-                'name': 'incorporationApplication',
-                'date': '2019-04-08',
-                'certifiedBy': 'full name',
-                'email': 'no_one@never.get',
-                'filingId': 1,
-                'effectiveDate': '2019-04-15T00:00:00+00:00'
+    'filing': {
```

Comment:
I think instead of using this we should be pulling the incorp filing from the example data of the schema (just so we're not updating this in 2 places) - I know we were using this before, but might be a good time to change it

Reply:
I just saw you actually did do this in the test